src/how-to-signer.md

```diff
@@ -1 +1,111 @@
 # How to operate a signer
+As a Stacker, you are responsible for ensuring the accuracy and integrity of incoming sBTC transactions. To satisfy this requirement and continue to earn BTC rewards, you must operate your own Signer or delegate your signing power to a third-party Signer. Below is a step-by-step guide to setting up and operating a Signer either by using the [default Signer implementation](#setting-up-a-default-signer) or by [utilizing the Signer SDK](#custom-signer-implementation).
+
+## Prerequisites
+Rust. To install, please [follow these instructions](https://www.rust-lang.org/tools/install).  
+Git
+
+## Setting up a Default Signer
+1. First, clone the Stacks sBTC mono repository:  
+```console
+git clone https://github.com/Trust-Machines/stacks-sbtc.git
+```
+2. Next, navigate to the stacks-signer-mini directory:  
+```console
+cd stacks-sbtc/stacks-signer-mini
+```
+3. Checkout the appropriate release branch you wish to use if you are not using the default main branch
+```console
+git checkout main
+```
+4. Compile the signer binary:  
+Note the binary path defaults to `target/release/signer`.
+```console
+cargo build --release
+```
+
+5. Generate a key pair:  
+Create a cryptographic key pair (public key and private key) to uniquely identify yourself as a Signer.
+Safeguard your private key securely to prevent unauthorized access.
+
+6. Update the default config:  
+Insert your private key into the conf/signer.toml file and update any defaults configurations you like for auto signing transactions.
+
+7. Run the binary:  
+```console
+cargo run --config conf/signer.toml
+```
+8. Monitor Incoming Transactions:  
+The signer binary operates a web server/client and it can be navigated to by default at http://localhost:3000/. 
+Here you can see pending transactions and manually review and sign transactions that cannot be automatically signed on your behalf.
```

Comment:
What are the conditions under which a transaction would not be automatically signed? It would be good to know as a signer how often I could expect this to happen and why it might happen.

Reply:
These are based on the configuration file. I have added a new section to explain these options and linked to it in the "udpate config" section.